### ISSUE TITLE
Possible to overlap zones?

### ISSUE THREAD

**Issue by EmSixTeen** [sylvainpolletvillard/postcss-grid-kiss#26]
More a question than an issue, but is it possible to have zones overlap? Love the idea of this, but need to be able to have things overlap. :)

**Comment by sylvainpolletvillard** [comment]
No, that's a limitation of grid -kiss. I can't think of a syntax that would let you do that and still be understandable at first glance. Any suggestions ?
